wurst/file/FileIO.wurst

```diff
@@ -0,0 +1,109 @@
+package FileIO
+import public ChunkedString
+import AbilityIds
+import ErrorHandling
+
+/* 	FileIO for Patches 1.31 and up with chunked string support for big data.
+	Reads and writes using preload files by manipulating default ability tooltips.
+
+	Read a file:
+	>	let fileContent = new File("MyFileName.pld").readAsString()
```

Comment:
This API example doesn't give me any reason to want a class at all - why not just `readToString(string filename)`?